lessons/intro/async/index.md

```diff
@@ -368,7 +372,8 @@ Ověřit si, že je na to program připravený, je poměrně složité, a na zaj
 správné funkčnosti je potřeba zamykání či jiné techniky, které bývají relativně
 pomalé, a tak se jim programátoři snaží vyhnout.
 A chyby vzniklé nesprávným ošetřením přepínání vláken bývají složité na odhalení
-a vyřešení. Pokud jste absolvovali předmět BI-OSY, jistě víte, o čem mluvíme.
+a vyřešení.
+(Studenti ČVUT: pokud jste absolvovali předmět BI-OSY, jistě víte, o čem mluvíme.)
```

Comment:
Klidně bych to vypustil úplně. Říct na cvičení to můžu, v materiálech to být nemusí.